code/controllers/subsystem/rogue/devotion.dm

```diff
@@ -7,9 +7,9 @@
 
 #define CLERIC_REQ_0 0
 #define CLERIC_REQ_1 100
-#define CLERIC_REQ_2 250
-#define CLERIC_REQ_3 500
-#define CLERIC_REQ_4 750
+#define CLERIC_REQ_2 500
```

Comment:
highly NOT crazy about this. This will effect church and cleric right? Grinding out devotion is just tedious and encourages sitting there AFK to grind out your spells round start. Don't need to make this worse it's already a 20 min grind.

Reply:
I only made this change because I thought it was JUST cleric, we can totally undo this - It was only because levels 1 and 2 were now going to be the *only* progression for clerics, so I thought I'd stretch it out.

Reply:
yeah I double checked they use the exact same req stuff (pretty sure). Tbh? Even if it WAS just cleric? It's about fun. Slapping annoying grinds with no meaningful gameplay  loop is not really a healthy way to nerf something.